.travis.yml

```diff
@@ -74,22 +74,17 @@ jobs:
       - make pylint
       - make mypy
 
-  - name: "Compile and test. Python 3.9"
+  - name: "Compile and test. Python 3.8"
     <<: *job_compile_common
     stage: Test multiple python versions
-    python: "3.9"
+    python: "3.8"
 
-  - name: "Compile and test. Python 3.10"
+  - name: "Compile and test. Python 3.9"
     <<: *job_compile_common
     stage: Test multiple python versions
-    python: "3.10"
```

Comment:
Why do you want to test it also for 3.9 only?  Can we stay on 3.10 for that?

Reply:
Done.